pkg/parsers/operatingsystem/operatingsystem_windows.go

```diff
@@ -62,3 +62,10 @@ func GetOperatingSystemVersion() (string, error) {
 func IsContainerized() (bool, error) {
 	return false, nil
 }
+
+// IsWindowsClient returns true if the SKU is client. It returns false on
+// Windows server, or if an error occurred when making the GetVersionExW
+// syscall.
```

Comment:
```suggestion
// IsWindowsClient returns true if the SKU is client. It returns false on
// Windows server.
```

Reply:
Ah, yes, I guess that's no longer relevant; updated 👍